shared/teams/channel/create-channels.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react'
+import * as Kb from '../../common-adapters'
+import * as TeamsTypes from '../../constants/types/teams'
+import * as TeamsGen from '../../actions/teams-gen'
+import * as Container from '../../util/container'
+import * as Styles from '../../styles'
+import CreateChannels from '../new-team/wizard/create-channels'
+
+type Props = Container.RouteProps<{teamID: TeamsTypes.TeamID}>
+
+export default (props: Props) => {
```

Comment:
`export default (props: Props) =>` makes the component anonymous in the debugging screens. Prefer 
```
const CreateChannels = (props: Props) => ...

export default CreateChannels
```